### PR TITLE
IGAPP-994: Implement rtl for pois route

### PR DESCRIPTION
**Pull request by f1sh1918**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

http://localhost:9000/testumgebung/ar/locations

1. Test MapView Controls for RTL
2. Desktop: Check Panel: Icons & content in POI Detail View, ListItems in PoiList
3. Mobile: Check BottomActionSheet PoiDetails and PoiList

### CODE REVIEW

**Review by klinzo — COMMENTED** [review]
~~I think lots of these adjustments should not be necessary as most of the things seems to be aligned perfectly well without the direction property (with the exception of the icons, i.e. back arrow, previous/next arrows, external link icon). Or am I missing something here?~~

File: web/src/components/PoiPanelNavigation.tsx [hunk 1]
```diff
@@ -21,19 +23,28 @@ const Label = styled.span`
...
   width: 16px;
   height: 14px;
   flex-shrink: 0;
   padding: 0 8px;
   object-fit: contain;
   align-self: center;
+  ${props =>
+    props.direction === 'rtl' &&
+    css`
+      transform: rotate(180deg);
```
Comment (klinzo) [inline web/src/components/PoiPanelNavigation.tsx:36]: Why not the following as well?
```suggestion
      transform: scaleX(-1);
```

File: web/src/components/PoiDetails.tsx [hunk 2]
```diff
@@ -21,20 +22,32 @@ const DetailsContainer = styled.div`
...
   width: 16px;
   height: 14px;
   flex-shrink: 0;
   padding: 0 8px;
   object-fit: contain;
   align-self: center;
+
+  ${props =>
+    props.direction === 'rtl' &&
+    css`
```
Comment (klinzo) [inline web/src/components/PoiDetails.tsx:35]: Maybe we could create a helper for this as this is used multiple times?

**Review by klinzo — APPROVED** [review]
Tested on firefox

**Comment by SteffiStuffel** [comment]
Can not test currently, as there are now pois for no language

**Comment by klinzo** [comment]
> Can not test currently, as there are now pois for no language

@SteffiStuffel you can use the production cms by entering `window.localStorage.setItem('api-url', 'https://cms.integreat-app.de')` in the console to test the pois.

**Comment by sarahsporck** [comment]
Currently not testable as the cityModel doesn't have a bounding box. Maybe we should set a default bounding box around Germany if there is none.

**Review by SteffiStuffel — APPROVED** [review]
Looks good :tada: 
tested on chrome

**Pull request merged**